package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "4.0.19",
+  "version": "4.0.20",
```

Comment:
Sure we can change to 4.1
Not so sure about the latter.

Reply:
whatever came out of that versioning discussion awhile back in RC?

Reply:
> whatever came out of that versioning discussion awhile back in RC?

I think Thor agreed it was a good idea, but we did not create a plan to do anything.

![image](https://user-images.githubusercontent.com/10002889/144499693-45e91fd1-ef03-48fd-bc25-68a4a6772869.png)

I really want to up-version our UIs to something "reasonable"... IMHO, 29 updates on Create UI doesn't accurately reflect all the changes we've added to it, and v0.x for Edit UI makes it look like non-production-ready software.

Cam, can you create a discussion in RC to move this forward? Doesn't feel big enough for a new ticket?

Reply:
the date is an interesting one as you can see at a glance when the UI was last deployed.  kinda helpful.  but i'm still so used to semantic versioning.  I'm good with either way.

Reply:
Suppose we go with 4.1.0 for this project... What will do to up-version the other UIs? I'm hoping for consistency...

Reply:
I agree the Date-versioning is more intuitive, although what happens to consistency if we change now. 
Where are our current versions used?
I can only think a few places
- Our version icon on the various applications.
- Our Releases
- Our past tags (that are usually in correlation with our version number)

Reply:
If we change THIS project now then we won't have to go and change it later :)  (except we will if we touch it again)

The other projects can be changed next time we touch them.

I think you're correct about the places that version is used.

Reply:
Seems like it's big enough to be it's own task, if we can all agree to a new approach.
Would be a new standard i think, so anywhere we version, we would need to update. 
This might be more than we think at first glance considering backend versions as well as shared components and whatnot.

Reply:
I think the date version if growing on me as I think about it.  Then we don't have to think about the major, minor and patch.  And you don't get into that issue where you have stuff at `v0.6.22` and worrying it looks non-production ready.

What's your take @severinbeauvais ?  Any preference for versioning format?  I know we agree they should be aligned across everything BE and FE.

Reply:
More i think about Shared Components though, i'm not sure how that would work, as Lerna handles those versions. 
Possibly configurable?

Reply:
oh yeah lerna.

Reply:
We can simply say that Lerna/shared components should use semantic versioning.

Date versioning is the way a lot of things are going. It feels like a big change. I dunno.

If we go with a new ticket, will that facilitate the discussion that we need to have? Do we still need to agree on something, or just do it? I think a RC discussion should suffice.

Reply:
@cameron-freshworks If you can hold off merging this PR until we decide today then you can be the first to make this change. Otherwise just merge it and we'll deal with it later (but soon I hope).

Reply:
As discussed in Standup, don't let versioning hold you up.

Reply:
As discussed in Stand Up, we will visit this topic on Monday to include Thor.